net-p2p/xmr-stak/files/xmr-stak

```diff
@@ -0,0 +1,28 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms fo the GNU General Public License, v2
+# /etc/init.d/xmr-stak
+name="xmr-stak"
+description="Unified all-in-one Monero miner"
+
+depend() {
+	need net
+}
+
+start() {
+	ebegin "Starting xmr-stak..."
+	start-stop-daemon --start --exec /usr/bin/xmr-stak \
```

Comment:
Please convert to `start_stop_daemon_args` usage. I.e. get rid of the start function. Also provide global `pidfile` variable to allow for OpenRC's status function.

Reply:
I also wasn't aware of that... https://wiki.gentoo.org/wiki/Handbook:X86/Working/Initscripts should be updated to show that form.